include/mlir/IR/OpImplementation.h

```diff
@@ -480,6 +495,14 @@ class OpAsmParser {
     result.append(types.begin(), types.end());
     return success();
   }
+
+private:
+  /// Parse either an operand list or a region argument list depending on
+  /// whether `operand' is true.
+  ParseResult parseOperandOrRegionArgList(SmallVectorImpl<OperandType> &result,
+                                          bool operand,
```

Comment:
I'd change 'operand' to 'isOperandList' to be more descriptive.

Reply:
Done.